src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -296,12 +296,12 @@ public static void onLivingSetAttackTarget(LivingEntity entity, LivingEntity tar
     {
         MinecraftForge.EVENT_BUS.post(new LivingSetAttackTargetEvent(entity, target, targetType));
     }
-    
+
```

Comment:
Whitespace change

Reply:
I hate unnecessary whitespace being added, and also having to re-add unnecessary whitespace for the sole purpose of keeping changes in a PR minimal.